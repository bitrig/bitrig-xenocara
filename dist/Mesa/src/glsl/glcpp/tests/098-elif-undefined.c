#if 0
#elif UNDEFINED_MACRO
#endif
