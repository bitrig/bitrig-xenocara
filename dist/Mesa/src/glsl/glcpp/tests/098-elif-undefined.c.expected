0:2(22): preprocessor error: syntax error, unexpected IDENTIFIER
0:1(7): preprocessor error: Unterminated #if



